core/schema/directory.go

```diff
@@ -202,6 +202,8 @@ func (s *directorySchema) Install(srv *dagql.Server) {
 				dagql.Arg("target").Doc(`Location of the file or directory to link to (e.g., "/existing/file").`),
 				dagql.Arg("linkName").Doc(`Location where the symbolic link will be created (e.g., "/new-file-link").`),
 			),
+		dagql.NodeFunc("__gitIgnoreFor", DagOpWrapper(srv, s.gitIgnoreFor)).
```

Comment:
Maybe just call this `__gitignorePatterns`?

Just to confirm - this returns *docker*-style patterns right? After converting them from gitignore style patterns? If so, worth clarifying in `Doc`.

Reply:
Yes, applied in [7ff0141](https://github.com/dagger/dagger/pull/10853/commits/7ff0141df57caa39e12fadd9c25b19c5062431bb)